src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -447,6 +445,21 @@ public static function get(int $pageId, int $revisionId = null, string $language
         return $return;
     }
 
+    public static function isForbiddenToDelete(int $pageId): bool
+    {
+        return in_array($pageId, [1, 404], true);
```

Comment:
Wouldn't it be useful to explain that 1 is the homepage and 404 is the error-page?
And maybe store the "protected" pages in a static array in the class?
Or use constants: `HOME_PAGE_ID = 1, ERROR_PAGE_ID = 404`

Reply:
2 methods use the same values but only the 404 is never allowed to have children
fixed in 12022a46d